src/pages/home/index.tsx

```diff
@@ -1,15 +1,103 @@
+import ClassItem from '@/pages/home/components/ClassItem';
+import { genreWrapperStyle } from '@/pages/home/components/ClassItem/index.css';
+import DancerItem from '@/pages/home/components/DancerItem';
+import Footer from '@/pages/home/components/Footer';
+import GenreItem from '@/pages/home/components/GenreItem';
 import HomeHeader from '@/pages/home/components/HomeHeader';
+import {
+  containerStyle,
+  dancerListWrapperstyle,
+  deadlineClassWrapperStyle,
+  recommandClassWrapperStyle,
+} from '@/pages/home/index.css';
+import Flex from '@/components/Flex';
+import Head from '@/components/Head';
 import { useIntersect } from '@/utils/useIntersect';
+import { DANCERLIST, GENRELIST, RECOMMAND_CLASSLIST } from '@/constants/home';
 import { vars } from '@/styles/theme.css';
 
 const Home = () => {
   const [targetRef, isVisible] = useIntersect(false);
-
   return (
-    <div style={{ height: '300rem' }}>
+    <>
       <div ref={targetRef} style={{ height: '37.5rem', backgroundColor: vars.colors.gray02 }}></div>
-      <HomeHeader isVisible={isVisible}></HomeHeader>
-    </div>
+      <HomeHeader isVisible={isVisible} />
+
+      <div className={recommandClassWrapperStyle}>
+        <Head level="h2" tag="h4">
+          이 클래스는 꼭 들어야 해요!
+        </Head>
+        <Flex tag="ul" gap="0.8rem" marginTop="2rem" className={containerStyle}>
+          {RECOMMAND_CLASSLIST.map((data) => (
+            <ClassItem
+              key={data.lessonId}
+              lessonId={data.lessonId}
+              lessonImageUrl={data.teacherImageUrl}
+              lessonLevel={data.lessonLevel}
+              lessonGenre={data.lessonGenre}
+              lessonName={data.lessonName}
+              teacherNickname={data.teacherNickname}
+              teacherImageUrl={data.teacherImageUrl}
+              lessonStartDateTime={data.lessonStartDateTime}
+              lessonEndDateTime={data.lessonEndDateTime}
+              lessonStreetAddress={data.lessonStreetAddress}
+            />
```

Comment:
밑에도 동일한 ClassItem 컴포넌트가 재사용되는 것 같아요. 취향 차이이긴한데,, 전달 되어야 하는 props가 많으니 따로 변수로 선언해서 동시에 같이 관리하면 가독성이나 유지보수도 좋을 것 같습니다!!
```suggestion

 const createLessonData = (data) => ({
    lessonId: data.lessonId,
    lessonImageUrl: data.teacherImageUrl,
    lessonLevel: data.lessonLevel,
    lessonGenre: data.lessonGenre,
    lessonName: data.lessonName,
    teacherNickname: data.teacherNickname,
    teacherImageUrl: data.teacherImageUrl,
    lessonStartDateTime: data.lessonStartDateTime,
    lessonEndDateTime: data.lessonEndDateTime,
    lessonStreetAddress: data.lessonStreetAddress,
  });
  
<ClassItem key={data.lessonId} {...lessonData} />;```

Reply:
여기 api가 나눠질수도 있고 정확히 같은 데이터는 아닐 수 있어서 일단 이렇게 해두고 추후에 변경 예정입니다! 
희선님 의견도 참고해서 변경할게요 감사합니다~!!